CHANGES.md

```diff
@@ -1,5 +1,28 @@
 # Changes
 
+## edge-22.1.4
+
+This edge release features a new configuration annotation, support for
+externally hosted Grafana instances, and other improvements in the CLI,
+dashboard and Helm charts. 
+
+* Added a new annotation to configure skipping subnets in the init container
+  (`config.linkerd.io/skip-subnets`). This configuration option is ideal for
+  Docker-in-Docker (dind) workloads (thanks @michaellzc!)
+* Added support in the dashboard for externally hosted Grafana instances
+  (thanks @jackgill!)
+* Introduced resource block to `linkerd-jaeger` Helm chart (thanks
+  @yuriydzobak!)
+* Introduced parametrized datasource (`DS_PROMETHEUS`) in all Grafana
+  dashboards
```

Comment:
... "this allows pointing to the right Prometheus datasource when importing a dasbhoard"